src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchArgs.cs

```diff
@@ -40,7 +44,53 @@ public int VerifyInt(string number, int defaultValue)
                 return verifiedNumber;
             }
 
-            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalid_number, number));
+            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalidOptionValue, number, option));
+        }
+
+        public bool VerifyFormat(string format)
```

Comment:
I think the name of this method is a bit confusing because its functionality is to check if the format is JSON or not. I propose using `IsJsonFormat` as a slightly better choice.